javers-core/src/test/groovy/org/javers/core/cases/CaseInheritanceWithIgnoredField.groovy

```diff
@@ -0,0 +1,50 @@
+package org.javers.core.cases
+
+import jakarta.persistence.Id
+import org.javers.core.JaversBuilder
+import org.javers.core.metamodel.annotation.DiffIgnore
+import org.javers.core.metamodel.annotation.TypeName
+import org.javers.core.metamodel.type.EntityType
+import spock.lang.Specification
+
+/**
+ * @author khan
+ */
+class CaseInheritanceWithIgnoredField extends Specification {
+    @TypeName("A")
+    class A {
+        @Id
+        String id
+        String field
+    }
+    @TypeName("B")
+    class B extends A {
+        @DiffIgnore
+        String someOtherField
+    }
+    @TypeName("C")
```

Comment:
not sure what's the difference between B and C. Do we need both these classes in this test?

Reply:
ok, the order of calls to getTypeMapping() is importany, got it